docs/src/content/docs/ko/guides/i18n.mdx

```diff
@@ -0,0 +1,218 @@
+---
+title: 국제화 (i18n)
+description: 여러 언어를 지원하도록 Starlight 사이트를 구성하는 방법을 알아보세요.
+---
+
+import FileTree from '../../../../components/file-tree.astro';
+
+Starlight는 라우팅, 대체 콘텐츠 및 전체 RTL(오른쪽에서 왼쪽으로) 언어 지원을 포함하여 다국어 사이트에 대한 기본 지원을 제공합니다.
+
+## i18n 구성
+
+1. Starlight 통합에 [`locales`](/ko/reference/configuration/#locales) 및 [`defaultLocale`](/ko/reference/configuration/#defaultlocale)을 전달하여 사이트가 지원해야 할 언어에 대해 알려주세요.
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: '나의 문서',
+   			// 사이트의 기본 언어를 영어로 설정합니다.
+   			defaultLocale: 'en',
+   			locales: {
+   				// 영어 문서는 `src/content/docs/en/`에 있습니다.
+   				en: {
+   					label: 'English',
+   				},
+   				// 중국어 간체 문서는 `src/content/docs/zh/`에 있습니다.
+   				zh: {
+   					label: '简体中文',
+   					lang: 'zh-CN',
+   				},
+   				// 아랍어 문서는 `src/content/docs/ar/`에 있습니다.
+   				ar: {
+   					label: 'العربية',
+   					dir: 'rtl',
+   				},
+   			},
+   		}),
+   	],
+   });
+   ```
+
+   `defaultLocale`은 대체 콘텐츠 및 UI 라벨에 사용되므로 콘텐츠 작성을 시작할 가능성이 가장 높거나 이미 콘텐츠가 있는 언어로 선택하세요.
+
+2. `src/content/docs/`에 각 언어에 대한 디렉토리를 만듭니다. 예를 들어 위에서 사용한 구성의 경우 다음과 같이 디렉토리를 생성할 수 있습니다.
+
+   <FileTree>
+
+   - src/
+     - content/
+       - docs/
+         - ar/
+         - en/
+         - zh/
+
+   </FileTree>
+
+3. 이제 언어 디렉토리에 콘텐츠 파일을 추가할 수 있습니다. 여러 언어로 작성된 페이지를 연결하기 위해 같은 이름의 파일을 사용하면 Starlight의 모든 i18n 기능(대체 콘텐츠, 번역 알림 등)을 활용할 수 있습니다.
+
+   예를 들어, 아랍어와 영어 홈페이지를 각각 나타내려면 `ar/index.md` 및 `en/index.md`를 만듭니다.
+
+### 루트 로케일 사용
+
+경로에 i18n 접두사가 없는 언어를 제공하려면 "루트" 로케일을 사용할 수 있습니다. 예를 들어 영어가 루트 로케일인 경우 영어 페이지 경로는 `/en/about` 대신 `/about`가 됩니다.
+
+루트 로케일을 설정하려면 `locales` 구성에서 `root` 키를 사용하세요. 루트 로케일이 콘텐츠의 기본 로케일이기도 한 경우 `defaultLocale`을 제거하거나 값을 `root`로 설정하세요.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+	integrations: [
+		starlight({
+			title: '나의 문서',
+			defaultLocale: 'root', // 선택적
+			locales: {
+				root: {
+					label: 'English',
+					lang: 'en', // 루트 로케일은 lang이 반드시 필요합니다.
+				},
+				zh: {
+					label: '简体中文',
+					lang: 'zh-CN',
+				},
+			},
+		}),
+	],
+});
+```
+
+루트 로케일을 사용하는 경우 해당 언어로 작성된 페이지를 전용 언어 폴더 대신 `src/content/docs/`에 직접 보관하세요. 예를 들어 위 구성을 사용할 때 영어와 중국어에 대한 홈 페이지 파일은 다음과 같습니다.
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - **index.md**
+      - zh/
+        - **index.md**
+
+</FileTree>
+
+#### 단일 언어를 사용하는 사이트
+
+기본적으로 Starlight는 단일 언어(영어) 사이트입니다.
+다른 언어로 단일 언어 사이트를 만들려면 `locales` 구성에서 `root`를 해당 언어로 설정하세요.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+	integrations: [
+		starlight({
+			title: '나의 문서',
+			locales: {
+				root: {
+					label: '简体中文',
+					lang: 'zh-CN',
```

Comment:
While usually we want translations to match the English version 1:1, I think this is a good example where we might want to deviate to show Korean here (assuming many Korean readers who want a monolingual site, may want that to be Korean).

What do you think?

```suggestion
					label: '한국어',
					lang: 'ko',
```